XCUITests/SettingsTest.swift

```diff
@@ -22,38 +22,33 @@ class SettingsTest: BaseTestCase {
         navigator.goto(SettingsScreen)
         let appsettingstableviewcontrollerTableviewTable = app.tables["AppSettingsTableViewController.tableView"]
         appsettingstableviewcontrollerTableviewTable.cells["OpenWith.Setting"].swipeUp()
+        waitforExistence(appsettingstableviewcontrollerTableviewTable.staticTexts["Use Compact Tabs"])
         appsettingstableviewcontrollerTableviewTable.staticTexts["Use Compact Tabs"].swipeUp()
+        waitforExistence(appsettingstableviewcontrollerTableviewTable.staticTexts["Passcode"])
         appsettingstableviewcontrollerTableviewTable.staticTexts["Passcode"].swipeUp()
         appsettingstableviewcontrollerTableviewTable.staticTexts["Show Tour"].swipeUp()
-        wait(for: 5)
+        wait(for: 2)
+        waitforExistence(appsettingstableviewcontrollerTableviewTable.staticTexts["Help"])
         let helpMenu = appsettingstableviewcontrollerTableviewTable.cells["Help"]
         XCTAssertTrue(helpMenu.isEnabled)
         helpMenu.tap()
-        
-        waitForValueContains(app.textFields["url"], value: "support.mozilla.org")
+            waitForValueContains(app.textFields["url"], value: "support.mozilla.org")
         waitforExistence(app.webViews.staticTexts["Firefox for iOS"])
         XCTAssertTrue(app.webViews.staticTexts["Firefox for iOS"].exists)
-        
```

Comment:
we can leave the existing empty lines, because they separate between different steps, making it easier to read.  Same for below deletion of empty lines